notifications_utils/formatters.py

```diff
@@ -602,25 +607,41 @@ def link(self, link, title, content):
 )
 
 
+def escape_lang_tags(_content: str) -> str:
```

Comment:
I think this is a much better approach than what we had previously. This way the markup parser will leave the language tags alone and we don't have to remove random `<p>` tags after the fact.